third_party/mini_chromium/CMakeLists.txt

```diff
@@ -66,17 +72,17 @@ list(APPEND MINI_CHROMIUM_SOURCES
 )
 
 if(NOT MINGW)
-    list(APPEND MINI_CHROMIUM_SOURCES
+    mc_append_sources(
         strings/utf_string_conversion_utils.cc
     )
 else()
-    list(APPEND MINI_CHROMIUM_SOURCES
-        ../../utf_string_conversion_utils.mingw.cc
+    mc_append_sources(
+        ${CMAKE_CURRENT_SOURCE_DIR}/../../utf_string_conversion_utils.mingw.cc
```

Comment:
Yes. There is no problem with paths because the target is only modified in the `mini_chromium` directory and that function is defined inside the correct folder.


The problem you were having in sentry-native was that the target was created in `dir1` and you added sources to it relative to `dir2`. The make system could not find these latter sources because it was looking for them relative to `dir1`.

Reply:
Or was it this warning?
https://dev.azure.com/sentry-builds/sentry-native/_build/results?buildId=1872&view=logs&j=20bc0e3d-29cc-5b49-42c5-dfbddf1762eb&t=53b537ca-00de-5a61-6445-73a526fb28b6&l=82